templates/style/_navbar.scss

```diff
@@ -5,15 +5,15 @@ div.nav-container {
     // Nothing is supposed to be over or hovering the top navbar. Maybe add a few others '('? :)
     z-index: 999;
     height: $top-navbar-height;
-    border-bottom: 1px solid $color-border;
-    background-color: #fff;
+    border-bottom: 1px solid var(--color-border);
+    background-color: var(--color-background);
```

Comment:
What's the difference between `$color-border` and `var(--color-border)`? If the change is only because you renamed them in `_themes.scss`, I think I'd prefer to have the old name.

Reply:
`$color-border` is an SCSS variable, so gets converted into a hard-coded color in the CSS generated. `var(--color-border)` is a CSS variable applied by the browser at runtime, so when we change the theme it changes the value of `--color-border` on the `html` element, which cascades down to all the other elements and updates any styles using `var(--color-border)`.